src/patternfly/components/Alert/alert.scss

```diff
@@ -121,24 +141,28 @@
     --pf-c-alert__title--Color: var(--pf-c-alert--m-info__title--Color);
 
     // Set border left color in case it is also an inline alert
-    --pf-c-alert--m-inline--BorderLeftColor: var(--pf-c-alert--m-info-inline--BorderLeftColor);
+    --pf-c-alert--m-inline--before--BackgroundColor: var(--pf-c-alert--m-inline--m-info--before--BackgroundColor);
   }
 
   &.pf-m-inline {
     --pf-c-alert--BoxShadow: var(--pf-c-alert--m-inline--BoxShadow);
     --pf-c-alert__title--Color: var(--pf-c-alert--m-inline__title--Color);
     --pf-c-alert__icon--BackgroundColor: var(--pf-c-alert--BackgroundColor);
-    --pf-c-alert--m-success__icon--Color: var(--pf-c-alert--m-success-inline__icon--Color);
-    --pf-c-alert--m-danger__icon--Color: var(--pf-c-alert--m-danger-inline__icon--Color);
-    --pf-c-alert--m-warning__icon--Color: var(--pf-c-alert--m-warning-inline__icon--Color);
-    --pf-c-alert--m-info__icon--Color: var(--pf-c-alert--m-info-inline__icon--Color);
-
-    border-color: var(--pf-c-alert--m-inline--BorderColor);
-    border-style: var(--pf-c-alert--m-inline--BorderStyle);
-    border-width: var(--pf-c-alert--m-inline--BorderWidth);
-    border-left-color: var(--pf-c-alert--m-inline--BorderLeftColor);
-    border-left-style: var(--pf-c-alert--m-inline--BorderLeftStyle);
-    border-left-width: var(--pf-c-alert--m-inline--BorderLeftWidth);
+    --pf-c-alert--m-success__icon--Color: var(--pf-c-alert--m-inline--m-success__icon--Color);
+    --pf-c-alert--m-danger__icon--Color: var(--pf-c-alert--m-inline--m-danger__icon--Color);
+    --pf-c-alert--m-warning__icon--Color: var(--pf-c-alert--m-inline--m-warning__icon--Color);
+    --pf-c-alert--m-info__icon--Color: var(--pf-c-alert--m-inline--m-info__icon--Color);
+    --pf-c-alert--before--BackgroundColor: var(--pf-c-alert--m-inline--before--BackgroundColor);
+
+    border-top: var(--pf-c-alert--m-inline--BorderTopWidth) solid var(--pf-c-alert--m-inline--BorderColor);
+    border-right: var(--pf-c-alert--m-inline--BorderRightWidth) solid var(--pf-c-alert--m-inline--BorderColor);
+    border-bottom: var(--pf-c-alert--m-inline--BorderBottomWidth) solid var(--pf-c-alert--m-inline--BorderColor);
+    border-left: var(--pf-c-alert--m-inline--BorderLeftWidth) solid var(--pf-c-alert--m-inline--BorderColor);
```

Comment:
Since you don't have unique border colors (and I don't think you need them), you could write this shorter like:

```
    border: solid var(--pf-c-alert--m-inline--BorderColor);
    border-width: var(--pf-c-alert--m-inline--BorderTopWidth) var(--pf-c-alert--m-inline--BorderRightWidth) var(--pf-c-alert--m-inline--BorderBottomWidth) var(--pf-c-alert--m-inline--BorderLeftWidth);
```